aspnetcore/blazor/fundamentals/configuration.md

```diff
@@ -19,7 +19,29 @@ Blazor WebAssembly loads configuration from:
 * App settings files by default:
   * `wwwroot/appsettings.json`
   * `wwwroot/appsettings.{ENVIRONMENT}.json`
-* Other [configuration providers](xref:fundamentals/configuration/index) registered by the app. Not all providers are appropriate for Blazor WebAssembly apps. Clarification on which providers are supported for Blazor WebAssembly is tracked by [Clarify configuration providers for Blazor WASM (dotnet/AspNetCore.Docs #18134)](https://github.com/dotnet/AspNetCore.Docs/issues/18134).
+* Other [configuration providers](xref:fundamentals/configuration/index) registered by the app. Not all providers or provider features are appropriate for Blazor WebAssembly apps:
+  * [Azure Key Vault configuration provider](xref:security/key-vault-configuration): The provider isn't supported for managed identity and application (client) ID with client secret scenarios. Application ID with a client secret isn't recommended for any ASP.NET Core app, especially Blazor WebAssembly apps because the client secret can't be secured on the client to access to the service.
```

Comment:
Maybe clarify here that the "client" is an end-user's browser?

Reply:
Yes, but perhaps "client-side" for that last mention because it might not involve a browser.

Also, I'll change ...

> application (client) ID

... to ...

> application ID (client ID)

... which seems clearer to me.